docs/getting-started/sync-with-github.md

```diff
@@ -1,61 +1,181 @@
+
+![Amplication service dashboard with "Commit changes & build" button outlined.](./assets/sync-with-git-provider/commit-changes-and-build.png)
+
+:::tip
+For an example of changes you can make to your service, see [Building New Versions of Your Service](/building-new-versions/).
+:::
+
+To view your previous commits for a project, select the drop down at the top of the page, and click `Commits`.
+
+![Amplication service dashboard with "Commits" button outlined.](./assets/sync-with-git-provider/commits-dropdown.png)
+
+You'll now see a list of your commits for the project.
+You can go to a specific commit by clicking on it in the sidebar.
+Each commit will contain changes, if any, for all services.
+
+![](./assets/sync-with-git-provider/930b3e58-62e3-4b14-950c-13e11a41c878.png)
+
+Now click on the specific service that you're interested in seeing the build for.
+A build contains a log of all the updates and changes that were made to that specific service during that commit.
+
+To view the Pull Request in GitHub, select the most recent commit and click `Open With GitHub` in the footer.
+
+You can now view the code changes generated by Amplication, and merge them.
+
+Now let's review what happens from GitHub's side when Amplication generates a new Pull Request on your repository.
+
+## Merging Amplication's Pull Request into your branch
+
+Amplication triggers the creation of a commit on the changed files whenever a new build is performed on a project.
+The changed files are all the files that were added, removed, or changed from the last build.
+
+![A pull request generated by Amplication on GitHub](./assets/sync-with-git-provider/github-pull-request-amplication-branch.png)
+
+Amplication reduces the chances of user changes being overridden by interacting with the user’s repository through an `amplication` branch. This branch is created in your selected repository by Amplication's GitHub App.
+
+:::important
+If you're on the [Free Plan](/sync-with-git-differences-between-plans), Amplication will create multiple `amplication` branches and pull requests for each commit.
+The format will be `amplication-build-{id}`.
+
+If you're on the paid plan, Amplication will instead manage all commits on a single branch called `amplication`.
+:::
+
+Let's review what's happening here in its entirety.
+
+## The `amplication` branch
+
+Users can merge Amplication-made changes by merging pull requests created by Amplication from the `amplication` branch to the user repository's default branch.
+
+## The commit message
+
+The title of the pull request and the commit message will both come from the message you enter into the commit message box in Amplication's pending changes section.
+
+Also included is a link to your build highlighted in a codeblock. If you click on that link you'll be taken back to Amplication to see the build log for this specific commit.
```

Comment:
we recently removed the code block and the link will just be presented to the user as text.